src/blocks/page.css

```diff
@@ -0,0 +1,33 @@
+@font-face {
+  font-family: "Roboto-regular";
+  src: url(../fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Roboto-medium";
+  src: url(../fonts/Roboto-Medium.ttf);
+  font-style: normal;
+}
```

Comment:
looks like this font face doesn't work

Reply:
check font on demo and at mockup

Reply:
Inspect tool in Chrome shows both fonts do work.